src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -1813,6 +1813,11 @@ public StreamPendingSummary build(Object data) {
 
       List<Object> objectList = (List<Object>) data;
       long total = BuilderFactory.LONG.build(objectList.get(0));
+
+      if (total == 0) {
+        return new StreamPendingSummary(total, null, null, null);
```

Comment:
We just don't return null on our own.
We return null or whatever only after reading it.

Reply:
Check #3793